iTjuana/itj-library
# :lock: Add authorization to manage page

### Description of the Change


Only Admin users can enter to manage page

### Verification Process


With admin account
1. Run the UI
2. Must be able to see manage on the navbar
3. Go to manage page and should be able to see data


With user account
1. Run the UI
2. Shouldn't be able to see manage on the navbar
3. Go to manage page and should be redirected to index

### Applicable Issues


- Closes #97 

### Release Notes


Add Authorization to manage and roles
